### PR TITLE
Changes made for RHDEVDOCS-1688

### PR DESCRIPTION
**Pull request by nkakkar81**
Following change have been made:
1. Added a new file for Installation of Pipelines Operator. This file will be added to the OCP bucket, once the folder structure is in place. I have used attributes for product names, which will be populated only when the file is moved to the OCP bucket.
2. Modified the Pipelines Resources file to add missing resource types - Storage and Cloud Event.

### CODE REVIEW

**Comment by nkakkar81** [comment]
@Preeticp , @joaedwar - Request you to please review this PR.

File: modules/ROOT/pages/op-installing_pipelines.adoc [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+// Installing Openshift Pipelines Operator
+[id="installing-pipelines-operator_{context}"]
+== Installing the {op-title} Operator
+
+The {op-title} Operator can be installed using the {product-title} instructions for installing Operators.
+
+You can install the {op-title} Operator in the host cluster by following the {product-title} instructions on installing an Operator.
+
+For more details on installation instructions, see the {product-title} documentation on xref:../operators/olm-adding-operators-to-cluster.adoc#olm-adding-operators-to-a-cluster[adding Operators to a cluster].
+
```
Comment (Preeticp) [inline modules/ROOT/pages/op-installing_pipelines.adoc:14]: The 3 sentences on L 13, 15 and 17 say the same thing, please combine them into one with the link.

File: modules/ROOT/pages/op-installing_pipelines.adoc [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+// Deleting Openshift Pipelines Operator
+[id="deleting-openshift-pipelines-operator_{context}"]
+== Deleting the {op-title} Operator
+
+You can remove the {op-title} Operator from the host cluster by following the {product-title} instructions on deleting an Operator.
+
+For details, see the {product-title} documentation on xref:../operators/olm-deleting-operators-from-cluster.adoc#olm-deleting-operators-from-a-cluster[deleting Operators from a cluster].
```
Comment (Preeticp) [inline modules/ROOT/pages/op-installing_pipelines.adoc]: Same thing here, combine to make one sentence.

**Comment by nkakkar81** [comment]
@Preeticp  - Comments Incorporated. Thank!

**Pull request merged**